source/_meta/_00-head.mustache

```diff
@@ -4,14 +4,13 @@
     <title>{{ title }}</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
-    
+    <script src="https://use.fontawesome.com/a9bf041a16.js"></script>
     <link rel="stylesheet" href="../../css/style.css?{{ cacheBuster }}" media="all" />

```

Comment:
the href needs to change to href="../css/style.css?{{ cacheBuster }}" right now its not pointing to the right file